flashinfer/gdn_kernels/blackwell/gated_delta_net_chunked.py

```
@@ -1289,9 +1265,34 @@ def _cg(num_threads):
                         state_inp_ready_producer,
                         shared_inp_ready_producer,
                         o_store_producer,
-                    ),
-                    (True, 0, head_idx),
-                )
+                        checkpoint_idx,
```

Ruff (0.15.15)

[warning] Unpacked variable `checkpoint_idx` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)